loki/alerting-rules.yaml

```diff
@@ -6,34 +6,43 @@ metadata:
     app.kubernetes.io/name: loki
 spec:
   groups:
-    - name: loki.alerts
+    - name: loki_alerts
       rules:
-      - alert: LokiRequestErrors
-        annotations:
-          message: |
-            {{ $labels.job }} {{ $labels.route }} is experiencing {{ $value | humanizePercentage }}% percent errors.
-        expr: |
-          100 * sum(rate(loki_request_duration_seconds_count{status_code=~"5.."}[1m])) by (namespace, job, route)
-            /
-          sum(rate(loki_request_duration_seconds_count[1m])) by (namespace, job, route)
-            > 10
-        for: 15m
-        labels:
-          severity: critical
-      - alert: LokiRequestPanics
-        annotations:
-          message: |
-            {{ $labels.job }} is experiencing {{ $value | humanizePercentage }}% percent increase of panics.
-        expr: |
-          sum(increase(loki_panic_total[10m])) by (namespace, job) > 0
-        labels:
-          severity: critical
-      - alert: LokiRequestLatency
-        annotations:
-          message: |
-            {{ $labels.job }} {{ $labels.route }} is experiencing {{ $value | humanize }}s 99th percentile latency.
-        expr: |
-          namespace_job_route:loki_request_duration_seconds:99quantile{route!~"(?i).*tail.*"} > 1
-        for: 15m
-        labels:
-          severity: warning
+        - alert: LokiRequestErrors
+          annotations:
+            message: |
+              {{ $labels.job }} {{ $labels.route }} is experiencing {{ printf "%.2f" $value }}% errors.
+          expr: |
+            100 * sum(rate(loki_request_duration_seconds_count{status_code=~"5.."}[1m])) by (namespace, job, route)
+              /
+            sum(rate(loki_request_duration_seconds_count[1m])) by (namespace, job, route)
+              > 10
+          for: 15m
+          labels:
+            severity: critical
```

Comment:
can we add service_owner too?

Reply:
This is a public kustomize base; if BitGo wants to add a service_owner it would go in an overlay/patch